beacon-chain/sync/validate_beacon_attestation.go

```diff
@@ -36,10 +36,14 @@ func (s *Service) validateCommitteeIndexBeaconAttestation(ctx context.Context, p
 	ctx, span := trace.StartSpan(ctx, "sync.validateCommitteeIndexBeaconAttestation")
 	defer span.End()
 
+	if msg.Topic == nil {
```

Comment:
lets put this all in our pubsub validation wrapper. All these general checks are cleaner to perform over there.

Reply:
Where? Is that? I thought checking msg.Topic == nil is an added check and very cheap, but I would be happy to add it to another place so we are 100% sure that the topic is set.

Reply:
https://github.com/prysmaticlabs/prysm/blob/master/beacon-chain/sync/subscriber.go#L176

You can just check it here and wrap it for all the topics

Reply:
Perfect, i'll add that in addition to the existing checks